modules/virt-customizing-storage-profile.adoc

```diff
@@ -58,53 +45,10 @@ metadata:
 spec:
   claimPropertySets:
   - accessModes:
-    - ReadWriteOnce <1>
+    - ReadWriteOnce
     volumeMode:
-      Filesystem <2>
-status:
+      Filesystem
+  status:
   provisioner: <unknown_provisioner>
   storageClass: <unknown_provisioner_class>
 ----
-<1> The `accessModes` that you select.
-<2> The `volumeMode` that you select.
-+
-After you save your changes, the selected values appear in the storage profile `status` element.
```

Comment:
Should we keep this? Maybe we can move it to the `virt-customizing-storage-profile-default-cloning-strategy.adoc` module?

Reply:
I removed only the last sentence, as I wasn't sure what it meant.